### ISSUE TITLE
[A11Y] Use underline for links (not just color)

### ISSUE THREAD

**Issue by bwbroersma** [bikkelhart/internet-nl#68]
https://dev3.internet.nl/site/b6a.nl/31/#siteipv6

![Image](https://github.com/user-attachments/assets/74356e8d-cf2b-4e0e-b030-e1b7e398516b)

Not sure: but note that not having an underline, the text color contrast difference probably becomes [SC 1.4.11](https://w3.org/TR/WCAG21#non-text-contrast) - [Important visual information](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) that should be **3**:1. It currently is green-400 `#037584` to black-200 `#191919`: [**3.24**:1](https://webaim.org/resources/contrastchecker/?fcolor=037584&bcolor=191919)), so it would pass (at first I thought it would not). And for dark mode green-300 `#8BF1FF` to white-100 `#FFF`: [**3**:1](https://webaim.org/resources/contrastchecker/?fcolor=8BF1FF&bcolor=FFFFFF).

https://github.com/bikkelhart/internet-nl/blob/1c80edf9c94f6968bacb1f8a276cc7997db2cea7/frontend/css/variables.css#L181

**Comment by bikkelhart-menno** [comment]
Added a underline for all anchors in paragraph content and article content in [42214526b431280587cadfd5493377784e0a7dca](https://github.com/bikkelhart/internet-nl/commit/42214526b431280587cadfd5493377784e0a7dca)
